components/provisioning/org.wso2.carbon.identity.provisioning.connector.google/src/main/java/org/wso2/carbon/identity/provisioning/connector/google/GoogleConnectorConstants.java

```diff
@@ -26,6 +26,9 @@
     public static final String ATTRIBUTE_PRIMARYEMAIL = "primaryEmail";
 
     public class PropertyConfig {
+
+        private PropertyConfig(){}
```

Comment:
format code with rearrange entries ticked, this will move constructor etc. in correct order